### DISCUSSION TITLE
Feature Resquest - input_datetime

### DISCUSSION THREAD

**Discussion by pgale** [Clooos/Bubble-Card#611] [Feature requests]
Keeping the bubble card theme, it would be great to have an input_datetime (time only) selector. I've only found one card that has that so far but it's not very stylish or configurable. Having something in a single height bubble card style would be great. Might have to pop up the controls for hour, minute, seconds to set them. I guess date would also be useful for some.

**Comment by poachedindividual** [comment]
I completely agree, I am surpirsed at the lack of cards with nice date time selectors. 
This would work well with the new climate card for setting auto on and off timers. 
Would love to see this in a future release.
